resources/configurations/examples/ros_discovery_client.yaml

```diff
@@ -53,9 +45,9 @@ ClientROS2:                                                       # 5
 #
 # The last one will act as a Discovery Client.
 # It will connect to a remote Discovery Server and communicate to every node connected to it, and route data
-# from the other interfaces.
+#  from the other interfaces.
```

Comment:
I like this format as well, but we do not use it (try to convince @rsanchez15 )
```suggestion
# from the other interfaces.
```

Reply:
:+1: Changed in 145bc84c186a1d673f7ef83ee3308e58c264d503